app/main.py

```diff
@@ -1,2 +1,63 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
+    key_hash: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` should be a float to ensure proper calculation. Change `THRESHOLD = 2 / 3` to `THRESHOLD = 2 / 3.0` or `THRESHOLD = 0.666...`.